### PR TITLE
Add TL specific rules to the style guide

### PR DESCRIPTION
**Pull request by obeid**
I would love extra eyes on this @kaezarrex

### CODE REVIEW

File: linters/.eslintrc [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
 // Copy this file, and add rule overrides as needed.
 {
-  "extends": "airbnb"
+  "extends": "airbnb",
+  
+  "rules": {
+    // Override any settings from the "parent" configuration
+    "no-unused-vars": true,
+    "indent": [2, 2], //TODO: look into adding custom indentation for switch case statments
+    "max-len": [2, 99, 2, {"ignoreComments": true, "ignoreUrls": true}]
```
Comment (zhaostu) [inline linters/.eslintrc]: I thought we were using max length of 100.

**Comment by zhaostu** [comment]
@obeid It's really adding rules to the eslint configuration file, not the style guide.

**Comment by obeid** [comment]
@zhaostu You're right, style will be checked with jscs.

**Comment by kaezarrex** [comment]
@obeid looks good

**Pull request merged**